docs/DeveloperGuide.md

```diff
@@ -501,41 +501,102 @@ testers are expected to do more *exploratory* testing.
 ### 7.1 Launch and shutdown
 
 1. Initial launch
-
    1. Download the jar file and copy into an empty folder
+   2. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
 
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
+2. Saving window preferences
+   1. Resize the window to an optimum size. Move the window to a different location. Close the window.
+   2. Re-launch the app by double-clicking the jar file.<br>
+       Expected: The most recent window size and location is retained.
 
-1. Saving window preferences
+### 7.2 Adding a new student record
 
-   1. Resize the window to an optimum size. Move the window to a different location. Close the window.
+Prerequisites: Existing student records do not have the names or IDs that will be added.
+1. Test case: `addstud nm/Peter Tan id/452B class/1F`
+   
+Expected: A new student recordord with the provided details is added to the list. Details of the student record are shown in the status message. Since no exam grades have been provided, the student card UI does not show anything below the grades section.
```

Comment:
```suggestion
Expected: A new student record with the provided details is added to the list. Details of the student record are shown in the status message. Since no exam grades have been provided, the student card UI does not show anything below the grades section.
```